lib/node_modules/@stdlib/stats/base/varianceyc/examples/index.js

```diff
@@ -18,18 +18,12 @@
 
 'use strict';
 
-var randu = require( '@stdlib/random/base/randu' );
-var round = require( '@stdlib/math/base/special/round' );
-var Float64Array = require( '@stdlib/array/float64' );
+var uniform = require( '@stdlib/random/array/uniform' );
 var varianceyc = require( './../lib' );
 
-var x;
-var i;
-
-x = new Float64Array( 10 );
-for ( i = 0; i < x.length; i++ ) {
-	x[ i ] = round( (randu()*100.0) - 50.0 );
-}
+var x = uniform( 10, -50.0, 50.0, {
+	'dtype': 'float64'
```

Comment:
```suggestion
	'dtype': 'generic'
```

The original code generated a "generic" array.